### DISCUSSION TITLE
Map libre problem at JupyterLab

### DISCUSSION THREAD

**Discussion by ThDonato** [giswqs/geog-312#103] [Q&A] [answered]
Hello sir, I am following your tutorial on YouTube and I'm getting the following error message during import overture from maplibre (leafmap)
![image](https://github.com/user-attachments/assets/f6905085-effc-40b9-9a15-f812b85f16cc)

**Comment by giswqs** [comment, marked as answer]
```bash
%pip install maplibre
```
